.github/workflows/quality_control.yml

```diff
@@ -22,7 +22,10 @@ jobs:
           python-version: 3.11
       - name: "Install all libraries"
         run: |
-          pip3 install -r requirements.txt
+          pip3 install pytest pylint
+          for FILE in $(find . -name 'requirements.txt'); do
+            pip3 install -r $FILE
+          done
```

Comment:
Would pylint and pytest would both installed from requirements.txt anyway? not sure if you need to run `pip3 install pytest pylint`

Reply:
I was thinking that way we don't need to add pylint and pytest to our requirements files - as the project doesn't actually need them to run. Plus we don't have to repeat pylint and pytest in every requirements file.